Modifies conversion helper with default carrier data

lstellway opened the pull request:
Fix for Issue #8 

This update defaults the carrier **code** and **title** to the provided carrier code while creating a tracking number during the order update process (cron job).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Commit:
Modifies the conversion class to default to the provided carrier code for package data

Commit:
Fixes references to $code variable

Commit:
Fixes order items shipment value

Checks if the item is available to ship and pulls the quantity to ship (versus the overall quantity ordered)

loganstellway commented:
Also added a commit to check if order items can be shipped in `Amazon\MCF\Helper\Conversion::getAmazonItemsArrayFromRateRequest()`

Commit:
Updates getAmazonItemsArrayFromRateRequest()

The `getAmazonItemsArrayFromRateRequest` method is used for both quote items and order items. 

The commit adds a check for an order item, skips items if they cannot be shipped, and utilizes the `getQtyToShip` method when submitting orders to Amazon.
